app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,32 @@
+class OrdersController < ApplicationController
+  def show
+    @order = Order.find_by(id: session[:order_id])
+    render :show
+  end
+
+  def edit
+    @order = Order.find_by(id: session[:order_id])
+    @user = User.find_by(id: session[:user_id])
```

Comment:
You have a helper method in the Application controller that would help you here. Can you find it?